frontend/src/components/checkbox/index.js

```diff
@@ -1,28 +1,105 @@
 /** @jsx jsx */
 import PropTypes from 'prop-types'
 import { jsx, css } from '@emotion/core'
-import { StyledLabel } from './StyledLabel'
-import { StyledInput } from './StyledInput'
+import { Input } from '../input'
+import { Container } from '../container'
+import { Label } from '../label'
 import { UniqueID } from '../unique-id'
 import { fontSize, fontWeight, lineHeight, space, color } from 'styled-system'
+import { BREAKPOINTS } from '../../theme'
+
+const mq = BREAKPOINTS.map(bp => `@media (min-width: ${bp})`)
 
 export const Checkbox = ({ label, ...props }) => {
   return (
     <UniqueID>
       {id => {
         return (
-          <div
-            css={css`
-              display: block;
-              position: relative;
-              padding: 0 0 0 38px;
-            `}
+          <Container
+            display="block"
+            position="relative"
+            pl={6}
+            mt={[3, null, 5]}
           >
-            <StyledInput type="checkbox" {...props} id={id} />
-            <StyledLabel htmlFor={id} {...props}>
+            <Input
+              type="checkbox"
+              position="absolute"
+              left="0"
+              top="0"
+              zIndex="1"
+              m={0}
+              css={theme => ({
+                height: theme.checkboxes.size,
+                width: theme.checkboxes.size,
+                opacity: 0,
+                zoom: '1',
+                ':checked + span:after': {
+                  opacity: '1',
+                },
+                ':focus + span:before': {
+                  boxShadow: `0 0 0 3px ${theme.colors.yellow}`,
```

Comment:
I added a “focusColor” in there (which... is yellow) that you could use here instead.